link/kprobe_test.go

```diff
@@ -482,10 +482,6 @@ func TestKprobeOffset(t *testing.T) {
 					if errors.Is(tt.err, syscall.EILSEQ) && errors.Is(err, os.ErrNotExist) {
```

Comment:
What is the difference between EILSEQ and ERANGE? Should this also get the `sys.Error` treatment like `ERANGE`?

Reply:
EILSEQ is for an address which is not an instruction. ERANGE is for an sym+off that can't be resolved. As far as I could see only tracefs api would return ERANGE